modules/kuberay-cluster/values.yaml

```diff
@@ -95,17 +95,17 @@ head:
       # Ray recommends at least 8G memory for production workloads.
       memory: "8G"
       # Sum of ephemeral storage requests must be max 10Gi on Autopilot default class.
-      # This includes, ray-head, gcsfuse-sidecar, and fluent-bit.
-      ephemeral-storage: 4Gi
+      # This includes, ray-head, gcsfuse-sidecar, fluent-bit, and ray Autoscaler sidecar which requests 1Gi by default.
+      ephemeral-storage: 3Gi
```

Comment:
why change the storage value?

Reply:
The Ray Autoscaler sidecar has 1Gi storage requests so we need to decrease the storage requests for some other containers so to stay with the total 10Gi ephemeral storage requests limit on Autopilot default class. I made the decrease for ray-head and gcsfuse simply because the number works and I also want to leave some extra room. I added an in-line comments to explain it. i.e. line 98.